mitodl/mitxpro
# #743 Product page catalog details

#### Pre-Flight checklist


#### What are the relevant tickets?
#743 

#### What's this PR do?
Adds another `catalog_details` property to product page models so that marketing can keep the description on catalog cards and product pages independent. Also includes a migration step to copy all existing description into the new `catalog_details` field.

#### How should this be manually tested?
In the CMS you can use the edit page form to modify the catalog details value for any product page. Verify that same value is displayed on the catalog card for the product.

#### Screenshots (if appropriate)
![Screenshot from 2019-07-03 16-58-02](https://user-images.githubusercontent.com/45350418/60590038-6c68ea80-9db4-11e9-9a1e-69348b00d011.png)
![Screenshot from 2019-07-03 16-57-42](https://user-images.githubusercontent.com/45350418/60590039-6c68ea80-9db4-11e9-93c9-2b1ecf533272.png)
